common/scala/src/main/scala/whisk/core/entity/Identity.scala

```diff
@@ -37,27 +37,38 @@ object Identity extends MultipleReadersSingleWriterCache[Identity, DocInfo] with
     override def cacheKeyForUpdate(i: Identity) = i.authkey
     implicit val serdes = jsonFormat4(Identity.apply)
 
+    def get(datastore: AuthStore, namespace: EntityName)(
+        implicit transid: TransactionId): Future[Identity] = {
+        implicit val logger: Logging = datastore
+        implicit val ec = datastore.executionContext
+        val ns = namespace.asString
+
+        cacheLookup(ns, {
```

Comment:
As noted in a private chat: Can this cause problems with the `cacheKeyForUpdate` above?